tests/python/unittest/test_library_loading.py

```diff
@@ -17,10 +17,13 @@
 
 # This test checks if dynamic loading of library into MXNet is successful
 
-import mxnet as mx
 import os
+import platform
+import unittest
+import mxnet as mx
 from mxnet.test_utils import download
 
+@unittest.skipIf(platform.machine() not in ['x86_64', 'AMD64'], "not all machine types supported")
```

Comment:
suggest to check machine and OS, also factor out the check in a function.

Reply:
'AMD64' is the value given by Windows so checking OS is not required. Refactored the check into a function.